config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get "/customers/:id", to: "customers#show"
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  post "/customers/:customer_id", to: "customers#show"
```

Comment:
Consider: `post "/customers/:customer_id/items", to: "customer_items#create"`

Based on the comment above about what controller to route it. Also, this path that includes `/items` helps indicate that this path is going to associate a customer and item together